docs/max_time.rst

```diff
@@ -5,7 +5,8 @@ Maximum Execution Time
 ======================
 
 To ensure fairness, there is a maximum execution time for each Qiskit Runtime job. If
-a job exceeds this time limit, it is forcibly terminated. The maximum execution time is the
+a job exceeds this time limit, it is forcibly canceled. This is represented in the job
+status as `Canceled - Ran too long`. The maximum execution time is the
```

Comment:
With https://github.com/Qiskit/qiskit-ibm-runtime/issues/168 we will move away from the deprecated job status string and instead use the new `state` property

```
"state": {
    "status": "Cancelled",
    "reason": "Ran too long"
}
```

I assume this change will be transparent for users and we can keep it as-is here...

Reply:
Yes, since the UI will continue to present it as one thing.